### DISCUSSION TITLE
New installation: problem accessing some of the Tools section (NB fixed by changing to a simpler password)

### DISCUSSION THREAD

**Discussion by srchild** [mcguirepr89/BirdNET-Pi#739] [General]
New installation on a Raspberry Pi 400. 

I can enter the Tools section after answering the password challenge, and can access and edit settings

I can enter System Controls, Services, Custom Species List, Excluded Species list

I cannot enter System Info, File Manager, Database Manager, Web Terminal. These throw up a password dialog but the same user/password is not accepted.

Before I set a password for Tools section I was able to enter all sections, but the above problem arises now that I have set a password.

I have started again and done a full reinstall and the same thing happens again. Before I set a password I can access all parts of the Tools just by using the username birdnet with no password. Keeping the username the same but setting a password creates the problem above.

If I go into advanced settings and delete the password then I can once again enter all sections of the Tools menu!

I have another installation of Birdnet-Pi running on a RPi4. This is updated today to the latest version. It does not have this problem with the tools section.

Ahah! I just set the password on my new installation to the same password used by my first installation and it works properly. The difference? The password I was using earlier included a $ whereas the password which works properly is alphanumeric only. It seems that the Tools section password must not contain a $ symbol.

**Comment by ehpersonal38** [comment]
Yep this has been an issue for quite some time - https://github.com/mcguirepr89/BirdNET-Pi/issues/276.

I suppose at this point I should just add some code to the web interface that just rejects passwords with special characters. Good job troubleshooting this though ~

EDIT: Here's the fix https://github.com/mcguirepr89/BirdNET-Pi/commit/8491d216c48e12fc4e6617c3a743ed2ccdfc5178

**Comment by AdamM68** [comment]
I have similar problem - can not access System Info tab.
I did a new installation on new SD card (rpi 3b+) and copied files from previous installation - described in [https://github.com/mcguirepr89/BirdNET-Pi/wiki/Backup-and-Restore-the-Database](url)
files I copied/replaced :
		~/BirdNET-Pi/birdnet.conf
		~/BirdNET-Pi/apprise.txt
		~/BirdNET-Pi/scripts/birds.db
               bootfs/config.txt
system is working properly, I can see the database and old detections .... but I can not access System info tab i tools menu

Can someone help me ?   Thanks
